src/Helpers/LocalMachineHelper.php

```diff
@@ -199,22 +201,15 @@ protected function fixFilename($filename)
     /**
      * Returns a set-up process object.
      *
-     * @param string|array $cmd The command to execute
+     * @param string $cmd The command to execute
      * @return Process
      */
-    protected function getProcess($cmd)
+    protected function getProcess(string $cmd)
     {
-        $process = null;
-        if (is_string($cmd)) {
-            $process = Process::fromShellCommandline($cmd);
-        } elseif (is_array($cmd)) {
-            $process = new Process($cmd);
-        }
-        if (!$process) {
-            throw new \InvalidArgumentException('$cmd should be a string or an array.');
-        }
+        $process = Process::fromShellCommandline($cmd);
```

Comment:
Was going to ask why disallow arrays from being passed, but I looked and see that Process::fromShellCommandline only takes a string as first argument..